src/components/core/TruncatedText/TruncatedText.js

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { ellipsis } from 'polished';
 
 const Base = ({ is: Component, ...props }) => <Component {...props} />;
 
 /**
  * Truncate text with an ellipsis.
  */
-const TruncatedText = styled(Base)`
-	white-space: nowrap;
-	overflow: hidden;
-	text-overflow: ellipsis;
+const TruncatedText = styled(Base).attrs({
+	title: props => props.children,
+})`
+	${props => ellipsis(props.width)};
```

Comment:
I ended up using default width `100%` without providing the argument and letting the container element define the width.

Reply:
So you're suggesting to remove the width prop?